packages/squiggle-lang/__tests__/SqProject/SqProject_imports_test.ts

```diff
@@ -1,81 +1,221 @@
 import { SqProject } from "../../src/index.js";
+import {
+  buildNaiveLinker,
+  runFetchBindings,
+  runFetchResult,
+} from "../helpers/projectHelpers.js";
 
-describe("Parse imports", () => {
-  const project = SqProject.create({ resolver: (name) => name });
-  project.setSource(
-    "main",
+describe("Imports tests", () => {
```

Comment:
I think "describe" items shouldn't need the word "tests", as that's implied. Just do, 
```describe("Imports")```
Or more precise.